060-ModernGitHubDev/README.md

```diff
@@ -0,0 +1,58 @@
+# What The Hack - Modern development and DevOps with GitHub
+
+## Introduction
+
+Increasing developer velocity and implementing proper DevOps procedures is a focus of most organizations in today's world. GitHub offers a suite of tools for developers to streamline code creation, automate tasks, and ensure code security. In this challenge-based hack you'll explore how to implement processes
+
+## The scenario
+
+As part of a give-back campaign, your organization is supporting a local pet shelter by updating, deploying and managing a web application for listing pets available for adoption.
+
+## Learning Objectives
+
+This DevOps with GitHub hack will help you learn how to use:
+
+1. manage source control with GitHub
+1. contribute code without installing resources locally with GitHub Codespaces
+1. gain the support of an AI pair programmer with GitHub Copilot
+1. automate deployment with GitHub Actions
+1. ensure code security with GitHub Advanced Security
+
+## Challenges
+
+- [Challenge 0](./Student/challenge00.md) - Setup and introduction
+- [Challenge 1](./Student/challenge01.md) - Configure your development environment
+- [Challenge 2](./Student/challenge02.md) - Add a feature to the existing application
+- [Challenge 3](./Student/challenge03.md) - Setup continuous integration and ensure security
+- [Challenge 4](./Student/challenge04.md) - Create a deployment environment
+- [Challenge 5](./Student/challenge05.md) - Setup continuous deployment
+
+
+## Prerequisites
+
+- Your own Azure subscription with **owner** access. See considerations below for additional guidance.
+- A GitHub Enterprise account
```

Comment:
This was originally built for an enterprise customer, thus the callout. It could be done on a public repository as well, but the repo would need to be public. The attendee would still need a GitHub Copilot account, however. I did update the content to add context about public repos